models/review_model.js

```diff
@@ -3,21 +3,24 @@ const db = require("../db/connection");
 exports.fetchReviewById = (review_id) => {
   return db
     .query(
-      `SELECT * FROM reviews
-      WHERE review_id = $1`,
+  `SELECT reviews.review_id, title, review_body, designer, review_img_url, reviews.votes, category, owner, reviews.created_at FROM reviews
+  JOIN comments ON reviews.review_id = comments.review_id
+  WHERE reviews.review_id = $1`,
       [review_id]
     )
     .then((results) => {
+      console.log(results.rows)
       if (!results.rows.length) {
         return Promise.reject({ status: 404, msg: "Not found" });
       }
+      results.rows[0].comment_count = results.rows.length
```

Comment:
this is much better. You may struggle doing this for future comment_counts though. It might be worth looking at `GROUP BY` as a nice refactor.